src/components/Card.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useEffect } from "react";
+import { IoIosAddCircle } from "react-icons/io";
+import { useNavigate } from "react-router-dom";
+
+const Card = () => {
+  const [data, setData] = useState([]);
+  const navigate = useNavigate();
+  function handleDelete(email) {
+    const items = JSON.parse(localStorage.getItem("phoneBook"));
+    const index = items.indexOf(email);
```

Comment:
It's better to use optional chaining `?` while accessing an object property's or calling a function as It will short circuits to `undefined` if object is `undefined` or `null`. Starting using this practice in your code and as it will help you out in the future.